docs/self-managed/operational-guides/production-guide.md

```diff
@@ -0,0 +1,572 @@
+
+### Index Retention
+
+An index lifecycle management (ILM) policy in OpenSearch is crucial for efficient management and operation of large-scale search and analytics workloads. ILM policies provide a framework for automating the management of index lifecycles, which directly impacts performance, cost efficiency, and data retention compliance.
+
+Here is how the ILM policy can be configured for the core component. This can be added to your `production-values.yaml`:
+
+```yaml
+core:
+  retention:
+    enabled: true
+    minimumAge: 30d
+    policyName: core-record-retention-policy
+```
+
+If you would like more information on configuring ILM policy. Please refer to [our guide](/self-managed/operate-deployment/data-retention.md).
+
+### Configuring Backups
+
+In order to configure backups, please refer to the [backup guide](/self-managed/operational-guides/backup-restore/backup-and-restore.md) for self-managed installations.
+
+## Scaling and Performance
+
+At this point you should already have a solid base to run your platform in a production setting. The rest of this guide gives you general Kubernetes based guidance on configuration of the Camunda Helm Chart for long-term maintenance.
+
+Here are some points to keep in mind when considering scalability:
+
+- To scale the Core component, the following values can be modified:
+
+```yaml
+core:
+  clusterSize: "3"
+  partitionCount: "3"
+  replicationFactor: "3"
+```
+
+The `core.clusterSize` refers to the amount of brokers, the `core.partitionCount` refers to how [zeebe partitions](/docs/components/zeebe/technical-concepts/partitions.md) are configured for each cluster, and the `core.replicationFactor` refers to the [number of replicas](/docs/components/zeebe/technical-concepts/partitions.md/#replication) that each partition replicates to.
+
+:::note
+The `core.partitionCount` does not support dynamic scaling. You will not be able to modify it on future upgrades.
+:::
+
+- Check the resource (CPU and memory) limits set and make sure they are appropriate for your workload size. For example, the resource limits can be changed for the core component by modifying the following values:
+
+```yaml
+core:
+  resources:
+    requests:
+      cpu: 800m
+      memory: 1200Mi
+    limits:
+      cpu: 2000m
+      memory: 1920Mi
+```
+
+- It is possible to set a LimitRange on the namespace. Please refer to the [Kubernetes documentation](https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/) on setting a LimitRange.
+
+## Reliability Best Practices
+
+Here are some points to keep in mind when considering reliability:
+
+- Check node affinity and tolerations. Please refer to the [Kubernetes documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) to modify the node affinity and tolerations.
```

Comment:
Again, the why and maybe an example of how one should taint, node, zone, or something, and what we recommend as criteria.

Reply:
I included an example of the default setup. Thanks.